scripts/abins/input/abinitioloader.py

```diff
@@ -159,7 +163,7 @@ def _rearrange_data(self, data=None):
         atoms = abins.AtomsData(data["atoms"])
         return abins.AbinsData(k_points_data=k_points, atoms_data=atoms)
 
-    def save_ab_initio_data(self, data=None):
+    def save_ab_initio_data(self, data: dict) -> None:
```

Comment:
same as above

Reply:
As above: actually running the function with `None` will raise an error either way. This should make it clearer that you aren't supposed to do that!